### PR TITLE
docs: Document extensions [part1]

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -285,6 +285,251 @@ nativeWebTapStrict | boolean | See the description of the corresponding capabili
...
+
+#### Arguments
+
+Name | Type | Required | Description | Example
+--- | --- | --- | --- | ---
+element | string | yes | PickerWheel's internal element id (as hexadecimal hash string) to perform value selection on. The element must be of type XCUIElementTypePickerWheel | abcdef12-1111-2222-3333-444444
+order | string | yes | Either `next` to select the value next to the current one from the target picker wheel or `previous` to select the previous one. | next
+offset | number | no | The value in range [0.01, 0.5]. It defines how far from picker wheel's center the click should happen. The actual distance is calculated by multiplying this value to the actual picker wheel height. Too small offset value may not change the picker wheel value and too high value may cause the wheel to switch two or more values at once. Usually the optimal value is located in range [0.15, 0.3]. `0.2` by default | 0.15
+
+### mobile: handleAlert
```
Comment (KazuCocoa) [inline README.md:306]:
```suggestion
### mobile: alert
```

https://github.com/appium/appium-xcuitest-driver/blob/8632cc98e2eee7c5875d79f92ecf38cfdd42acf5/lib/commands/execute.js#L58

File: README.md [hunk 2]
```diff
@@ -285,6 +285,251 @@ nativeWebTapStrict | boolean | See the description of the corresponding capabili
...
+excludedAttributes | string | no | One or more comma-separated attribute names to be excluded from the XML output, thus only makes sense if `format` is set to `xml`. It might be sometimes helpful to exclude, for example, the `visible` attribute, to significantly speed-up page source retrieval. | visible,accessible
+
+#### Returned Result
+
+The page source tree formatted according to the given format argument.
+
+### mobile: getContexts
+
+Retrieves the list of available contexts including the extended context information, like urls and page names. This is different from the standard `getContexts` API, because the latter only has web view names without any additional information. In situation where multiple web views are available at once the client code would have to connect to each of them in order to detect the one, which needs to be interacted with. Although, this extra effort is not needed with the information provided by this extension.
+
```
Comment (KazuCocoa) [inline README.md:364]: Here has an argument

```
#### Arguments

Name | Type | Required | Description | Example
--- | --- | --- | --- | ---
waitForWebviewMs | number | no | Appium will wait for available WebViews up to the number | 5000
```

File: README.md (hunk 1 above)
Comment (mykola-mokhnach) [reply README.md:306]: fixed

File: README.md (hunk 2 above)
Comment (mykola-mokhnach) [reply README.md:364]: fixed

**Review by KazuCocoa — APPROVED** [review]
🚀

**Pull request merged**